website/docs/docs/dbt-versions/release-notes/08-May-2023/deployment-monitor.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Use the Deployment monitor page for greater visibility to your job run health"
+id: "deployment-monitor"
+description: "May 2023: "
+sidebar_label: "New: Deployment monitor visualizes your run health"
+sidebar_position: 5
+tags: [scheduler]
+---
+
+The new Deployment monitor helps you gain greater visibility into your deployment health and quickly detect any issues with the improved filtering and sorting capabilities. You can now easily curate the view based on environment, prioritizing the ones that matter most, such as Production. With pre-filtered lists for in progress and in queue activities, it's quicker and easier to track the relevant activity and stay on top of your deployment status.
+
+The Deployment monitor is accessible by going to the main dbt Cloud dashboard page or clicking the dbt logo on the upper-left. Some of the new and enhanced features includes:
+
+- A new run history visualization component to quickly scan a larger number of recent runs and get a better overview of overall health/activity.
+- Pre-filtered lists have been added for In Progress, In Queue, and Latest Failed jobs. 
```

Comment:
```suggestion
- Prefiltered lists have been added for in progress, in queue, and latest failed jobs. 
```

Reply:
maybe i should rephrase this so its clearer it describes the UI. so:

>Prefiltered views added to display In Progress, In Queue, and Latest Failed jobs.

![Screenshot 2023-05-31 at 17 05 51](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/834ae6be-54b6-4254-aa0b-9fe3b417b973)

Reply:
oh i get it, sounds good!! sugg using "**Latest Failed Runs**" since that's the exact UI label